analysis/import/CSVImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/csv/ParserDialog.kt

```diff
@@ -9,10 +9,20 @@ class ParserDialog {
     companion object : ParserDialogInterface {
 
         override fun collectParserArgs(): List<String> {
-            val inputFileName = KInquirer.promptInput(
-                    message = "What is the sourcemonitor CSV file that has to be parsed?",
-                    hint = "sourcemonitor.csv"
-            )
+            val inputFileNames = mutableListOf(KInquirer.promptInput(
+                    message = "Please specify the name of the first sourcemonitor CSV file to be parsed:",
+                    hint = "input.csv"
+            ))
+            while (true) {
+                val additionalFile = KInquirer.promptInput(
+                        message = "If you want to parse additional sourcemonitor CSV files, specify the name of the next file. Otherwise, leave this field empty to skip.",
+                )
+                if (additionalFile.isNotBlank()) {
+                    inputFileNames.add(additionalFile)
+                } else {
+                    break
+                }
```

Comment:
I know this is a bit late, but I personally would omit the else block by changing the condition to if additionalFile is blank > break